charts/zabbix/templates/statefulset-zabbix-proxy.yaml

```diff
@@ -79,10 +79,15 @@ spec:
           {{- end }}
           imagePullPolicy: {{ .Values.zabbixAgent.image.pullPolicy }}
           env:
+            {{- if .Values.zabbixAgent.ZBX_HOSTNAME }}
+            - name: ZBX_SERVER_HOST
+              value: {{ .Values.zabbixAgent.ZBX_HOSTNAMET | quote }}
+            {{- else }}
```

Comment:
Please fix this lines...

Code suggested:

`
            {{- if .Values.zabbixAgent.ZBX_HOSTNAME }}
            - name: ZBX_HOSTNAME
              value: {{ .Values.zabbixAgent.ZBX_HOSTNAME | quote }}
            {{- else }}
`